ledger/participant-integration-api/src/main/scala/platform/store/dao/events/package.scala

```diff
@@ -74,6 +75,7 @@ package object events {
       .map(_._1)
       .fold(Vector.empty[A])(_ :+ _)
       .concatSubstreams
+      .buffer(128, OverflowStrategy.backpressure)
```

Comment:
`.buffer(16)` did not bring much improvement. I did not investigate other buffer sizes.

Reply:
Can you pull this out into a constant somewhere? Or better yet, a parameter?

I worry about the volume of magic numbers inside our codebase. I'd really like to push them to at the very least, a single file with all the magic numbers.